packages/screenshots/Classes/Configuration/Configuration.php

```diff
@@ -270,6 +297,64 @@ public function createBasicConfig(): void
                             ['action' => 'click', 'link' => "List"],
                             ['action' => 'makeScreenshotOfFullPage', 'fileName' => "IntroductionDashboardFullPage"],
                         ],
+                        "startpage" => [
+                            [
+                                "action" => "setScreenshotsImagePath",
+                                "path" => "Images/AutomaticScreenshots/IntroductionPackage"
+                            ],
+                            [
+                                "action" => "setScreenshotsRstPath",
+                                "path" => "Images/AutomaticScreenshots/IntroductionPackage"
+                            ],
+                            [
+                                "action" => "resizeWindow",
+                                "width" => 1024,
+                                "height" => 512
+                            ],
+                            [
+                                "action" => "see",
+                                "text" => "Page"
+                            ],
+                            [
+                                "action" => "click",
+                                "link" => "Page"
+                            ],
+                            [
+                                "action" => "openPageTreePath",
+                                "path" => ["Congratulations"]
+                            ],
+                            ["action" => "switchToContentFrame"],
+                            [
+                                "action" => "waitForText",
+                                "text" => "Congratulations",
+                                "timeout" => 5
+                            ],
+                            [
+                                "action" => "makeScreenshotOfElement",
+                                "fileName" => "IntroductionBackend"
+                            ],
+
+                            [
+                                "action" => "resizeWindow",
+                                "width" => 1024,
+                                "height" => 2048
+                            ],
+                            [
+                                "action" => "amOnPage",
+                                "page" => "/?id=1"
+                            ],
+                            [
+                                "action" => "wait",
```

Comment:
Prefer `waitForText` or `waitForElement` over `wait` ..

Reply:
This is  a demontstration of how to take a screenshot of the frontend. It shouldnt depend on text to be found.